src/pyfaf/actions/create_problems.py

```diff
@@ -299,8 +294,7 @@ def _create_problems(self, db, problemplugin, #pylint: disable=too-many-statemen
             i = 0
             for db_report in db_reports:
                 i += 1
```

Comment:
In most cases like this, we could just do `for i, db_report in enumerate(db_reports, start=1)`.